plugin/proto/msg.proto

```diff
@@ -42,14 +42,13 @@ message BlockHeader {
 message EthereumConfig {
   bytes frontierBlockNumber = 1;
   bytes homesteadBlockNumber = 2;
-  bytes eip150BlockNumber = 3;
-  bytes eip160BlockNumber = 4;
```

Comment:
Removed this field because it wasn't used anywhere.

Reply:
Please leave the protobuf file alone in this PR. We want it to be backwards compatible.